hs_access_control/migrations/0021_auto_20170418_2042.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
@pkdash 
Is this hs_access_control migration file related to my changes to BaseResource class? or is this hs_access_control migration file just a long-missing migration file that randomly showed up this time because I ran "python manage.py migrate" that works on all apps.